tests/campaigns/test_integration.py

```diff
@@ -1,44 +1,76 @@
-from mock import patch
+from uuid import uuid4
 
-from base import CampaignTestCase
-import sitewit.services
 from sitewit.services import SitewitService
+from tests.base import SitewitTestCase
 
 
-class TestGetCampaign(CampaignTestCase):
+class BaseCampaignTestCase(SitewitTestCase):
+
+    non_existent_campaign_id = 23232323
+
+    @classmethod
+    def setUpClass(cls):
+        super(BaseCampaignTestCase, cls).setUpClass()
 
-    def setUp(self):
         service = SitewitService()
```

Comment:
Because property is on object level, but here we have `setUpClass()`, but anyway, probably we should just move SitewitService() to be class level attribute.